shell/platform/tizen/channels/text_input_channel.cc

```diff
@@ -563,9 +593,33 @@ void TextInputChannel::HideSoftwareKeyboard() {
           isSoftwareKeyboardShowing_);
   if (imfContext_ && isSoftwareKeyboardShowing_) {
     isSoftwareKeyboardShowing_ = false;
-    ecore_imf_context_reset(imfContext_);
-    ecore_imf_context_focus_out(imfContext_);
-    ecore_imf_context_input_panel_hide(imfContext_);
+
+    if (engine_->device_profile ==
+        "mobile") {  // FIXME : Needs improvement on other devices.
+      auto w = engine_->tizen_surface->GetWidth();
+      auto h = engine_->tizen_surface->GetHeight();
+
+      if (rotation == 90 || rotation == 270) {
+        engine_->SendWindowMetrics(h, w, 0);
+      } else {
+        engine_->SendWindowMetrics(w, h, 0);
+      }
+      engine_->tizen_surface->SetSize(w, h);
+      ecore_timer_add(
+          0.05,
```

Comment:
Is this a heuristic value? Is the delay needed?

Reply:
Yes, it is heuristic. I tested it on devices and emulators, it is better when I set some delay. it can be removed anytime. However, when using the timer, it looked better even with 0 value